lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4871,7 +4871,9 @@ namespace {
                                       OptionalAttr(/*implicit*/false));
       // FIXME: Handle IBOutletCollection.
 
-      if (overridden) {
+      // Only record overrides of class members.
+      if (overridden &&
+          overridden->getDeclContext()->getAsClassOrClassExtensionContext()) {
```

Comment:
I think this check should happen up in the initial search for overridden properties, not way after the fact.

Reply:
Done.